src/components/core/base/test/VisualizationLoadingHOC.spec.tsx

```diff
@@ -391,5 +391,17 @@ describe('VisualizationLoadingHOC', () => {
                 expect(onError).toHaveBeenCalledWith(new RuntimeError(ErrorStates.DATA_TOO_LARGE_TO_COMPUTE));
             });
         });
+
+        it('getPage should return promise if component has been unmounted', async () => {
```

Comment:
If possible, tweak the text little bit. It sounds like the function does not return promise when the component has not been unmounted.